recipes-connectivity/coredns/files/corefile

```diff
@@ -0,0 +1,14 @@
+.:EDGE_NODEDNSPORT {                   
+    log                        
```

Comment:
Remove the plugins `log`, `reload 1s` `cache 30` and `loadbalance`. They are handy for debugging, but not necessary in production. Explanation below:
- log: debug logs
- reload: automatically reloads the CoreDNS when changing the Corefile
- cache: cache results, since we only use CoreDNS for localhost lookups based on a hosts file, results will not be cached anyway
- load balance: The hosts file will only container 1-1 mapping between IP and hostname, so load balancing will be redundant